espnet2/bin/asr_inference.py

```diff
@@ -951,6 +1007,29 @@ def get_parser():
         default=False,
         help="Time synchronous beam search.",
     )
+    group.add_argument(
+        "--perutt_blist",
+        default="",
+        help="Per utterance biasing list",
+    )
+    group.add_argument(
+        "--biasinglist",
+        type=str,
+        default="",
+        help="Biasing list path",
+    )
+    group.add_argument(
+        "--bmaxlen",
+        type=int,
+        default=100,
+        help="maximum biasing list size",
+    )
+    group.add_argument(
```

Comment:
Do you use biasing list dropout during inference?

Reply:
I wanted to keep the option for using dropout during inference, but in my paper, I did not. The task of biasing is that we have a list of words/entities that we want the ASR to perform better on, so we normally assume we know what we want to bias. It is just a matter of how many distractors we get that influence the performance.